homeassistant/components/vera/__init__.py

```diff
@@ -151,6 +148,9 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
             hass.config_entries.async_forward_entry_setup(config_entry, platform)
         )
 
+    await hass.async_add_executor_job(controller.start)
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, controller.stop)
```

Comment:
We should wrap the `controller.stop` call inside a function that accepts the `Event` parameter from the Home Assistant event bus.

Reply:
👍